lib/ecto_enum.ex

```diff
@@ -79,6 +102,10 @@ defmodule EctoEnum do
           {:ok, @enum_map[int]}
         end
 
+        def load(string) when is_binary(string) do
+          {:ok, String.to_atom(string)}
```

Comment:
Instead of doing `String.to_atom(string)`, do the same as the code for loading an integer. It's faster to get from a map of values than doing `String.to_atom/1`.